hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -140,7 +140,12 @@ class HassioAddonConfig extends LitElement {
           ? {
               name: entry.name,
               required: entry.required,
-              selector: { number: { mode: "box" } },
+              selector: {
+                number: {
+                  mode: "box",
+                  step: entry.type === "float" ? "any" : undefined,
```

Comment:
Or I guess `entry.type === "float" ? "any" : 1`

Reply:
That works to, I just wanted to do as little here as possible and fall back to defaults instead.